packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -39,36 +51,40 @@ export function SignIn() {
         });
       }}
     >
-      <Heading level={1}>Sign in to your account</Heading>
+      <Heading level={1}>{translations.headerText}</Heading>
```

Comment:
This is another example where having the I18n + default text makes it easier to correlate what we see on the page with where it comes from in the code:

```suggestion
      <Heading level={1}>{I18n.get('Sign in to your account')}</Heading>
```

Real-world anecdote: when we ran into I18n issues last year, a customer would say "Resend Code isn't translated" (or something to that affect).  We'd open up the project, search "Resend Code", and immediately see "Ah! This isn't wrapped with `I18n`!"

It seems small, but keeping context can make these fixes into a 1-liner and find/replace-able.

Reply:
Ok, I'm fine if we want to do it like that. This pattern that I have in the PR is how we handled it in SageMaker and felt like it was cleaner. But I'm down to forego the `translations` object and just manually wrap each one right inside the HTML if we feel like it would help make things easier to debug.

Reply:
I appreciate that.  We went further than this last year (by even moving the object to yet another file) and when we removed code, we had no way of knowing how much (if any) of that abstracted object was still in use.

So that's where my mind is today – in 6 months, how can I find the relevant code ASAP and, if we delete some of this code, will I prevent unused code? (TypeScript definitions & CSS are great examples of de-coupled code becoming unused)